tensorflow/python/ops/array_ops.py

```diff
@@ -1107,7 +1107,8 @@ def concat(values, axis, name="concat"):
   Args:
     values: A list of `Tensor` objects or a single `Tensor`.
     axis: 0-D `int32` `Tensor`.  Dimension along which to concatenate. Must be
-      in the range `[-rank(values), rank(values))`.
+      in the range `[-rank(values), rank(values))`. If a negative number for
```

Comment:
I would reword this to emphasize that this is the same as indexing in Python.

Reply:
Thanks @caisq. The docstring has been updated.